cluster_advanced.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+<<<<<<< Updated upstream
```

Comment:
hi there! check your YAML header. also if you can delete the contents of the file and write a title and "Work in progress..."